Dockerfile

```diff
@@ -41,10 +41,17 @@ ARG HE_VERSION=
 RUN yum -y -q update && \
     yum -y -q clean all
 
+# Install python-pip
+RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
+RUN python get-pip.py
+
 # Add bundler-audit
 RUN gem install bundler-audit
 RUN bundle-audit update
 
+# Add safety
+RUN pip install safety
```

Comment:
Hey, can you do an install of a specific version? `safety==??` This means we would then be able to update the version of safety and it would correctly invalidate the docker layer cache.